src/javascript/app/base/binary_loader.js

```diff
@@ -151,7 +154,7 @@ const BinaryLoader = (() => {
             return;
         }
 
-        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] || '' });
+        const div_container = createElement('div', { class: 'logged_out_title_container', html: Client.isAccountOfType('financial') ? '' : content.getElementsByTagName('h1')[0] });
```

Comment:
the `|| ''` for the second condition should remain for pages that don't have header.